spec/build.xml

```diff
@@ -179,6 +179,22 @@
       </fileset>
     </schemavalidate>
   </target>
+  
+  <target name="validate-example-4" description="Validates Example 4.">
+    <!-- validate using RNC schema -->
+    <java fork="true" jar="${jing.jar}">
+      <arg value="-c"/>
+      <arg value="${rnc.schema}"/>
+      <arg value="${examples.dir}/ex4-overflow.xml"/>
+    </java>
+    <!-- validate using XSD schema -->
+    <schemavalidate fullchecking="true" warn="true">
+      <schema namespace="http://www.w3.org/ns/ttml" file="${xsd.schema}"/>
+      <fileset dir="${examples.dir}">
+        <include name="ex4-overflow.xml"/>
+      </fileset>
+    </schemavalidate>
+  </target>
 
   <target name="validate-examples"
 	    depends="validate-example-1, validate-example-2, validate-example-3"/>
```

Comment:
Need to add `validate-example-4` to the dependencies here.

Reply:
(also, separately, I see that the `build` target does not call `validate` first, which it should)